metadata.rb

```diff
@@ -751,8 +751,8 @@
           :description => "Hopsworks username",
           :type => "string"
 
-attribute "hops/jmx/adminPassword",
```

Comment:
Why remove the password and not just point it at the new attribute, so that cluster defns can override it?

Reply:
I put it back.